src/Model/Term.php

```diff
@@ -192,15 +192,9 @@ protected function init() {
 					return $queue;
 				},
 				'uri'                      => function () {
```

Comment:
@jeanfredrik for whatever reason, Github won't let me provide this suggestion inline, but after reviewing and hashing this out for a while, @josephfusco and I believe this should be updated to the following:

```php
'uri'                      => function () {
	$link = $this->link;

        // parse the link to ensure it's a valid url. 
	$maybe_url = wp_parse_url( $link );

	// If $maybe_url returns false, it means it isn't a valid URL, so we can assume
        // the link has been filtered upstream and we can return the link value.
	if ( false === $maybe_url ) {
		return $link;
	}

	// Replace the home_url in the link in order to return a relative uri.
	// For subdirectory multisites, this replaces the home_url which includes the subdirectory.
	return ! empty( $link ) ? str_ireplace( home_url(), '', $link ) : null;
},
```